spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ExampleHeader.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring;
+
+import javax.validation.constraints.NotNull;
+
+import com.google.common.base.MoreObjects;
+
+import com.linecorp.armeria.common.HttpHeaders;
+
+/**
+ * Used as an example header object in {@link AnnotatedServiceRegistrationBean}
+ * and {@link ThriftServiceRegistrationBean}.
+ */
+public final class ExampleHeader {
```

Comment:
`ExampleHeaders`? because it has `HttpHeaders` rather than an individual header.